src/DIPS.Xamarin.UI/Controls/Sheet/SheetBehavior.cs

```diff
@@ -620,9 +619,16 @@ private async Task TranslateBasedOnPosition(double newPosition)
                 return; //Return when we set property because it will lead to recursively calling this method
             }
 
-            if (Position > MaxPosition)
+            if (newPosition > MaxPosition) //If the content is to big
             {
-                Position = MaxPosition;
+                if (MinPosition > (double)MinPositionProperty.DefaultValue) //Check if MinPosition has been set

```

Comment:
just set to max? This doesn't make sense.

Reply:
Fixed, thanks <3